polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -103,42 +123,22 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageInt
 
   def withJep[T](t: => T): IO[T] = shift.evalOn(executionContext)(IO.delay(t))
 
-  private val jep = jepExecutor.submit {
+  protected val jep: Jep = jepExecutor.submit {
```

Comment:
Made this protected because in next python-related PR there will be a subclass of this for pyspark (kinda like `ScalaSparkInterpreter`) to handle the necessary py4j setup